pkgs/development/r-modules/README.md

```diff
@@ -79,6 +79,30 @@ environment, see `rstudioWrapper`, which functions similarly to
 Then like above, `nix-env -f "<nixpkgs>" -iA rstudioEnv` will install
 this into your user profile.
 
+Alternatively, if you would rather not polute your environement or want
+to keep all the settings within your project, you can create a custom
+`shell.nix` using the overlays functionality:
+
+```nix
+{ nixpkgsFunc ? import <nixpkgs>
+}:
+
+let
+  roverlay = self: super: {
+    rstudioEnv = super.rstudioWrapper.override {
+      packages = with self.rPackages; [ tidyverse ];
+    };
+  };
+
+  pkgs = nixpkgsFunc { overlays = [ roverlay ]; };
```

Comment:
Why do you define an overlay if all you ever use from the modified package set is just one attribute? That seems unnecessary.

Reply:
Looks like I need to properly read the manual before I submit changes ;) Sorry if it's not up to the standard. I will try to simplify it more. Do you think an example similar this should be in the manual at all?

Reply:
Ok, gotcha. Ineed it was way more complicated than it needed to be. Hope this is better.